src/elements/content-uploader/UploadInput.tsx

```diff
@@ -42,7 +45,16 @@ const UploadInput = ({
                 data-testid="upload-input"
                 directory={isFolderUpload ? '' : undefined}
                 multiple={isMultiple}
-                onChange={onChange}
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
That comes handy when trying to upload files and their size exceeded 50mb (50*1000*1000 as I figured Box doesn't like 50*1024*1024 too much ;)). 
Currnetly I handled it by observing classList string and reacting to the change after the `onBeforeUpload` fired and contentExplorer changed it's button classes etc.